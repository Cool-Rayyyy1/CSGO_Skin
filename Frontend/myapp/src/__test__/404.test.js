import React from 'react';
import renderer from 'react-test-renderer';
import NotFoundPage  from "../pages/404"


it ("snapshot testing for profile screen", () => {
    const tree = renderer.create(
        <NotFoundPage></NotFoundPage>
    ).toJSON();
    expect(tree).toMatchSnapshot();
})
