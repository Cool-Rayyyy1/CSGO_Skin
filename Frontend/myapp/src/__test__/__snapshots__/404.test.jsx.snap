// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot testing for profile screen 1`] = `
<div>
  <h2>
    404 Not Found!
  </h2>
</div>
`;
